src/resources/i18n/translations/en.json

```
@@ -186,7 +186,7 @@
   "ARIA LABEL - Queue Next": "Enqueue as next",
   "Audio livestreams": "Audio livestreams",
   "Auto": "Auto",
-  "Auto download by default on subscribe": "Turn on auto-downloading for a podcast when you subscribe to it.",
+  "Auto download by default on subscribe": "When subscribing.",
```

```suggestion
  "Auto download by default on subscribe": "Download podcasts once subscribed to.",
```
I think this is worse. Is it also clips?

---

It's not for clips. Resolving this suggestion to consolidate discussion on previous thread.